src/util/thpool/thpool.c

```diff
@@ -9,6 +9,8 @@
    ********************************/
 
 //#define _POSIX_C_SOURCE 200809L
+#include "../../config.h"
```

Comment:
I don't believe we need this include

Reply:
I think we need it because in jobqueue_init we call Config_Option_get